packages/ui/lib/configs/showcase.tsx

```diff
@@ -7,7 +8,11 @@ const SHOWCASE_HEADER = {
   showSocialIcons: true,
   actions: [
     {
-      content: <Button variant="outlined-primary">Launch App</Button>,
+      content: (
+        <Link href="https://app.unlock-protocol.com/locks">
```

Comment:
You might wanna use the AppConfig variable here so it can be changed via environment variable (would be needed for staging and master deployment change)